packages/docs/getting-started/running-a-validator.md

```diff
@@ -303,6 +314,13 @@ celocli account:balance $CELO_VALIDATOR_GROUP_ADDRESS
 celocli account:balance $CELO_VALIDATOR_ADDRESS
 ```
 
+If you only have enough balance in one of your accounts, you can transfer some Celo Gold using the following command:
```

Comment:
I think the faucet process will prevent this from happening, so I think we can drop this section